### DISCUSSION TITLE
Export Partial Hydration as an endpoint

### DISCUSSION THREAD

**Discussion by vish01** [Elderjs/elderjs#243] [Q&A] [answered]
How can I export partial hydration so I can use a component in a different App? I'm trying to compile Svelte components on-the-fly from App A to App B without packaging App A. Since Elderjs supports partial hydration, I think there should be a way to expose an endpoint like SvelteKit which can send `html`, `css` and `js` as a response so the caller can render the component in their DOM.
Any thoughts on this idea?
Any help is greatly appreciated!

**Comment by nickreese** [comment, marked as answer]
Hmm, not really the usecase. You can probably achieve by looking at the hydration related hooks but you'll need to fiddle with it quite a bit.

**Reply by vish01** [reply]
@nickreese Any specific hook you'd recommend to look into? I saw what's available and I can't figure which one is even relevant to what I want to do.

**Reply by nickreese** [reply]
@vish01 That is the thing, there isn't a hook to do what you want to do. Elder.js could be forked to do what you want to do or with a ton of fiddling you could get an outcome close to what you want, but it won't be easy or straightforward. It just isn't built to do what you're trying to do.
